staging/src/k8s.io/kubectl/pkg/cmd/testing/fake.go

```diff
@@ -499,7 +499,7 @@ func (f *TestFactory) UnstructuredClientForMapping(mapping *meta.RESTMapping) (r
 }
 
 // Validator returns a validation schema
-func (f *TestFactory) Validator(validate bool) (validation.Schema, error) {
+func (f *TestFactory) Validator(validateDirective cmdutil.ValidationDirective, verifier *resource.QueryParamVerifier) (validation.Schema, error) {
```

Comment:
Is this something you wanted to take in that pull-request?

Reply:
fixed